slopmop/cli/hooks.py

```
@@ -341,96 +440,150 @@ def _hooks_status(project_root: Path, hooks_dir: Path) -> int:
+    pre_push_file.write_text(
+        _generate_pre_push_hook_script(global_install=global_install)
+    )
     hook_file.chmod(
         hook_file.stat().st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH
     )
     pre_push_file.chmod(
         pre_push_file.stat().st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH
     )
 
+    if global_install:
+        subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)